deploy/bicep/groups/processing.bicep

```diff
@@ -92,6 +92,22 @@ module aksManagedIdentity '../modules/managed.identity.user.bicep' = {
   }
 }
 
+resource waitSection 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
+  kind: 'AzurePowerShell'
+  name: 'WaitSection'
+  location: location
+  properties: {
+    azPowerShellVersion: '3.0'
+    scriptContent: 'start-sleep -Seconds 300'
+    cleanupPreference: 'Always'
+    retentionInterval: 'PT1H'
+  }
+  dependsOn: [
+    aksManagedIdentity
+  ]
+}
+
+
```

Comment:
Are we waiting for the managed identity itself, or a role assignment? Is there a doc somewhere that says to wait for this amount of time? Seems like a long delay, so I'm just wondering if there's a source of truth on that.

Reply:
This document discusses the Service Principal propagation under "Symptom - Assigning a role to a new principal sometimes fails". I believe it applies to UAMI as well.

https://learn.microsoft.com/en-us/azure/role-based-access-control/troubleshooting?tabs=bicep#role-assignment-changes-are-not-being-detected

As for the exact timing, that may vary tenant to tenant. But I assume 5 minutes should enough time to propagate. If the user still faces this issue, we might have to increase the timing.

Reply:
Yeah, that's unfortunate there isn't a better solution right now. I would like to decrease the wait, not increase it, but oh well, seems like this is the best we can do for now.